views/partials/battles.handlebars

```diff
@@ -0,0 +1,22 @@
+          <div class="column is-4 is-multiline">
+            <div class="card">
+              <div class="is-64x64 px-6">
+                <img src=`{{{image_url}}}` alt="track image">
+              </div>
+              <div class="columns is-centered is-multiline">
+                <h1 class="title pt-2">Song Information</h1>
+                <div class="subtitle column is-6">
+                  <p>Title: {{{song_name}}}</p>
+                  <p>Artist: {{{song_artist}}}</p>
+                  <p>Score: {{{id}}} </p>
+                </div>
+                <div class="column is-6">
```

Comment:
new partial! that works but formatting of image url doesn't work. I think I need more tem.. literals.